waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/server/MetaStoreProxyServer.java

```diff
@@ -153,8 +156,30 @@ private void startWaggleDance(
       int maxWorkerThreads = hiveConf.getIntVar(ConfVars.METASTORESERVERMAXTHREADS);
       boolean tcpKeepAlive = hiveConf.getBoolVar(ConfVars.METASTORE_TCP_KEEP_ALIVE);
       boolean useFramedTransport = hiveConf.getBoolVar(ConfVars.METASTORE_USE_THRIFT_FRAMED_TRANSPORT);
+      boolean useSSL = hiveConf.getBoolVar(ConfVars.HIVE_METASTORE_USE_SSL);
 
-      TServerTransport serverTransport = tcpKeepAlive ? new TServerSocketKeepAlive(waggleDanceConfiguration.getPort())
+      TServerSocket serverSocket = null;
+
+      // enable SSL support for HMS
+      List<String> sslVersionBlacklist = new ArrayList<>();
```

Comment:
Is there any way at least some of this code could be factored into a separate private method? This method is getting quite long now...

Reply:
If we start tidying up the code I'll be harder to compare with the original class in future releases. I prefer to leave the code as is so we can pick copy easy copy bits of code

Reply:
OK, that makes sense, can you then add a comment above this code snippet saying which class in Hive this was copied from so it's easier to line them up in future?

Reply:
I've moved this to its own method as there are other pieces that were moved already